Source/Atomic/Input/Input.h

```diff
@@ -66,7 +63,7 @@ struct TouchState
     /// Finger pressure.
     float pressure_;
     /// Last touched UI element from screen joystick.
-    WeakPtr<UIElement> touchedElement_;
+    UIWidget* touchedWidget_;

```

Comment:
Maybe want a WeakPtr here? That way if the refcount goes to zero we'll have a null instead of dangling ptr?